dissect/target/plugins/child/hyper-v.py

```diff
@@ -17,8 +16,7 @@ class HyperVChildTargetPlugin(ChildTargetPlugin):
     PATH = "sysvol/ProgramData/Microsoft/Windows/Hyper-V/data.vmcx"
 
     def check_compatible(self):
-        if not self.target.fs.path(self.PATH).exists():
-            raise UnsupportedPluginError("No data.vmcx file found")
+        return self.target.fs.path(self.PATH).exists()
```

Comment:
Why this change?

Reply:
Consistency across all `child` plugins and because `_load_child_plugins()` in `target.py` checks boolean values instead of catching any `UnsupportedPluginError` exceptions.

Reply:
`UnsupportedPluginError` is a subclass of `PluginError` which is caught. In this case, raising an error might give a little bit more information on why it's not compatible.

Reply:
It already catches those exceptions though:
https://github.com/fox-it/dissect.target/blob/main/dissect/target/target.py#L345-L350

Although I agree the messaging could be improved (i.e. a try/except being moved closer to the `check_compatible` with a more descriptive message), the functionality is already the same.

Reply:
Thanks for explaining. Although the exception message will be 'broken child plugin' when in reality it's just a non-compatible child plugin.

Reply:
Please check https://github.com/fox-it/dissect.target/pull/148/commits/806d4c41b0890cc4c4af54d1955206f0106bd917. I've changed the logging around a bit for child plugins so it should be a lot cleaner and clear about what's happening.